scaffolding/src/commands.rs

```diff
@@ -32,7 +30,10 @@ impl CommandApplicator {
             match self.rx.try_recv() {
                 Ok(cmd)             => {
                     redraw = true;
-                    cmd.apply(&mut terminal).unwrap_or_else(exit_on_io_error);
+                    cmd.apply(&mut terminal).unwrap_or_else(|e| {
+                        panic!("Error applying command '{}' due to error: {}",
+                               cmd.repr(), e);
```

Comment:
Also, feel free to open a PR against my branch that directly handles the specific IO error you are seeing.